DigitalLearningSolutions.Data/Helpers/FilteringHelper.cs

```diff
@@ -229,6 +225,19 @@ private static IEnumerable<FilterOptionModel> GetFilterOptionsForPromptWithoutOp
             return options;
         }
 
+        private static string GetFilterPropertyNameForPrompt(Prompt prompt)

```

Comment:
As the value from this is put in the `Group` parameter of `BuildFilterValueString`, this should probably be called  `GetFilterGroupForPrompt`. Same should apply to the two methods it calls in the ternary.